packages/feedback-service/.env.example

```diff
@@ -0,0 +1,25 @@
+## Uncomment the lines as needed
+
+NAME=local
+NODE_ENV=local
+PRODUCTION=false
+
+## Server URLs
+API=https://localhost:8080
+GRAPHQL_API=https://localhost:8080/playground
```

Comment:
This url is invalid as we're not using GraphQL playground anymore.

Reply:
yes confirmed with Rigin so replacing with.
GRAPHQL_API=https://localhost:8080/graphql